spec/core/agent_plugin_manager_spec.rb

```diff
@@ -0,0 +1,36 @@
+# encoding: utf-8
+require "spec_helper"
+require "logstash/runner"
+require "stud/task"
+
+describe LogStash::AgentPluginManager do
+  class TestAgent < LogStash::Agent; end
+  class TestAgent2 < LogStash::Agent; end
+
+  subject { described_class }
+
+  after(:each) do
+    LogStash::AgentPluginManager.reset!
+  end
+
+  describe "the default registry with no registered plugins" do
+    it "should have the default agent" do
+      expect(subject.lookup(LogStash::AgentPluginManager::DEFAULT_AGENT_NAME)).to eql(LogStash::Agent)
```

Comment:
in order to make the lines smaller, I would break this one and have the lookup in a separate variable or even a ```let``` statement.